docs/index.rst

```diff
@@ -214,3 +283,100 @@ With the above report, Betelgeuse performs the following:
     test case in the jUnit XML file.  For this reason, it is highly recommended
     to run ``test-command`` command before ``test-run`` to make sure all
     required test cases are created/updated accordingly.
+
+.. _case_study:
+
+Case Study - A real world sample Test Case
+```````````````````````````````````````````
+Testimony tokens can be used to provide more information about a test case.  The
+more information one provides via these tokens, the more accurate the data being
+imported into Polarion.  For example:
+
+.. code-block:: python
+
+  import entities
+  import unittest
+
+  class EntitiesTest(unittest.TestCase):
+
+      def test_positive_create_user(self):
+      """Create a new user providing all expected attributes.
+
+      @Assert: User is successfully created 
+      @ID: 1d73b8cc-a754-4637-8bae-d9d2aaf89003
+      @Requirement: User Management
+      @CaseAutomation: Automated
+      @CaseLevel: Acceptance
+      @CaseComponent: CLI
+      @TestType: Functional
+      @CaseImportance: High
+      @Upstream: No
+      """
+          user = entities.User(name='David', age=20)
+          self.assertEqual(user.name, 'David')
+          self.assertEqual(user.age, 20)
+
+When the above test case is ran, Betelgeuse will make use of all 9 tokens
+provided and generates a more meaningful test case.
+
+Ok, this is cool. But wait, there is more! If you already read
+`Testimony documentation <http://testimony-qe.readthedocs.io/>`_, it supports
+tokens at different levels, namely:
+
+  - function level
+  - class level
+  - module level
+
+This feature can be leveraged to minimize the amount of information that needs
+to be written for each test case. Since most of the time, test cases grouped in
+a module usually share the same generic information, one could move most of
+these tokens to the ``module`` level and every single test case found by
+Betelgeuse will inherit these attributes. For example:
+
+
+.. code:: python
+
+    """
```

Comment:
I will be better to have a first line docstring, something like "Test cases for entities"